interface-definitions/include/accel-ppp/ppp-options-ipv6-interface-id.xml.i

```diff
@@ -0,0 +1,48 @@
+<!-- include start from accel-ppp/ppp-options-ipv6-interface-id.xml.i -->
+<leafNode name="ipv6-intf-id">
+  <properties>
+    <help>Fixed or random interface identifier for IPv6</help>
+    <completionHelp>
+      <list>random</list>
+    </completionHelp>
+    <valueHelp>
+      <format>random</format>
+      <description>Random interface identifier for IPv6</description>
+    </valueHelp>
+    <valueHelp>
+      <format>x:x:x:x</format>
+      <description>specify interface identifier for IPv6</description>
+    </valueHelp>
+  </properties>
+</leafNode>
+<leafNode name="ipv6-peer-intf-id">
+  <properties>
+    <help>Peer interface identifier for IPv6</help>
+    <completionHelp>
+      <list>random calling-sid ipv4</list>
+    </completionHelp>
+    <valueHelp>
+      <format>x:x:x:x</format>
+      <description>Interface identifier for IPv6</description>
+    </valueHelp>
+    <valueHelp>
+      <format>random</format>
+      <description>Use a random interface identifier for IPv6</description>
+    </valueHelp>
+    <valueHelp>
+      <format>ipv4</format>
+      <description>Calculate interface identifier from IPv4 address, for example 192:168:0:1</description>
+    </valueHelp>
+    <valueHelp>
+      <format>calling-sid</format>
+      <description>Calculate interface identifier from calling-station-id</description>
+    </valueHelp>
```

Comment:
missing regex constraint probing for `random`, `ipv4`, `calling-sid` and the IPv6 interface qualifier.

Reply:
Done!